mirage/views/utils/-private.ts

```diff
@@ -228,9 +227,8 @@ export function compare(
         return compareStrings(actualValue, comparisonValue, operator);
     } else if (typeof actualValue === 'boolean') {
         return compareBooleans(actualValue, queryParamIsTruthy(comparisonValue), operator);
-    } else {
-        throw new Error(`We haven't implemented comparisons with "${operator}" yet.`);
     }
+    throw new Error(`We haven't implemented comparisons with "${operator}" yet.`);
```

Comment:
Why are some of these errors not using i18n (not really in the review but didn't know these existed)?

Reply:
These are in mirage, which is only used in development and testing. I'm also not sure if mirage can get access to the i18n service (probably there's some way, but not necessary unless we want to i18n dev debugging messages).

Reply:
ahh fair enough - didn't noticed we passed into mirage code